nixos/modules/services/web-apps/movim.nix

```diff
@@ -179,6 +179,17 @@ let
       "mysql" = "mysql.service";
     }
     .${cfg.database.type};
+
+  # https://github.com/movim/movim/issues/314
+  movimCSP = ''"${
```

Comment:
This is going to produce a string whose first and last characters are literal quotation marks (`"`), right? But CSP headers don't have these characters in them (as far as I can tell from https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP, at least).

Is the quoting a h2o config/dsl thingy (the semantics of the `header.set` directive)? If it is, I suggest separating the CSP (which is a value that doesn't have quotes in it) from the "h2o config generation" to make it a little clearer what's going on.

(apologies if I'm misunderstanding this)